Tests/SpyableMacroTests/Extensions/UT_FunctionParameterListSyntax+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftSyntax
+import XCTest
+
+@testable import SpyableMacro
+
+final class UT_FunctionParameterListSyntaxExtensions: XCTestCase {
+  func testSupportsParameterTracking() {
+    XCTAssertTrue(
+      FunctionParameterListSyntax {
+        "param: Int"
+        "param: inout Int"
+        "param: @autoclosure @escaping (Int) throws async -> Void"
```

Comment:
Parser from `swift-syntax` is producing an error here:
<img width="758" alt="Screenshot 2023-12-29 at 8 32 11 PM" src="https://github.com/Matejkob/swift-spyable/assets/44930823/de43b3b8-ddc2-4ad7-b3e8-f7429a087a47">
This change should fix it:
```suggestion
        "param: @autoclosure @escaping (Int) async throws -> Void"
```

Reply:
I'm actually a little curious what tooling you used to catch this one. Is there a swift-syntax CLI tool that you can run against a project / directory?
It'd be cool to add whatever you did here to catch this to the Github Actions checks!

Reply:
I've just run unit tests in Xcode and `swift-syntax` library has logged this error into the console. I'm unsure why it did not throw an error when creating the node. I will investigate it later this week.